app/views/handlers/question.py

```diff
@@ -204,7 +204,7 @@ def evaluate_and_update_section_status_on_list_change(self, list_name):
         section_ids = self._schema.get_section_ids_dependent_on_list(list_name)
 
         section_keys_to_evaluate = (
-            self.questionnaire_store_updater.started_section_keys(
+            self._questionnaire_store.progress_store.started_section_keys(
```

Comment:
We should avoid using questionnaire store progress here as that technically should be private. It would be better to keep the method on questionnaire store updater, which calls the progress store one. We already do that pattern for a couple of the progress store methods.